### DISCUSSION TITLE
When do you plan to release pyspark 3.5.0?

### DISCUSSION THREAD

**Discussion by paulpaul1076** [stackabletech/spark-k8s-operator#324] [Q&A] [answered]
I see in stackable nexus, that spark 3.5.0 is available, but  pyspark 3.5.0 is not available. Could you please tell me when you plan to release it? Thanks!

**Comment by sbernauer** [comment, marked as answer]
Hi @paulpaul1076!

As you have noticed we have shipped Spark 3.5.0 in the Stackable 23.11 release.
In the [23.11 release notes](https://docs.stackable.tech/home/stable/release-notes#_release_23_11) we mention the following change

> [Spark: all product images contain pyspark](https://github.com/stackabletech/docker-images/pull/436)

So you should be able to use the `spark-k8s` images instead of the `pyspark-k8s` and your job should continue working
